examples/weather_bot/app/bot.py

```
@@ -0,0 +1,243 @@
+    query = BOT_PROMPT.format(user_query=input.query)
+    result: ReasoningResult = await asyncio.to_thread(
+        weather_agent.solve,
+        query,
+    )
+
+    return {"response": str(result.final_answer)}
+
+
+@app.get("/")
+async def root():
+    return {"message": "🤖 Hi! I’m WeatherBot. Use POST /ask with {query: 'your question'}"}
```

Ruff (0.14.11)

String contains ambiguous `’` (RIGHT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF001)